Twilio-org/phonebank
# [Ticket 125] Mutate volunteer call to connect contact

Manual Test:
  1. Register a user with your phone number
  2. Login as the user and join Campaign 1 from seed file
  3. Query database to get your user_id for use in the curl commands provided later.
  4. Run the following curl command to assign yourself a call:
```
curl -X POST \
  http://localhost:3000/users/<user id>/campaigns/1/calls \
  -H 'cache-control: no-cache' \
  -H 'postman-token: f284dab7-6744-a116-57b7-482331bd7efe'
```
  5. Run the following curl command and answer the call on your phone:
```
curl -X POST \
  http://localhost:3000/users/<user_id>/campaigns/1/calls/ \
  -H 'cache-control: no-cache' \
  -H 'postman-token: 5ce03639-4e27-ac5a-fe78-6ef21f481f33'
```
 6. Run the following curl command:
```
curl -X PUT \
  http://localhost:3000/users/3/campaigns/1/calls/2 \
  -H 'cache-control: no-cache' \
  -H 'content-type: application/json' \
  -H 'postman-token: e11cd22b-df6c-90b9-ae7f-3f8efcfb85a1' \
  -d '{
    "status":"IN_PROGRESS"
}'
```
7. observe response with status 200 and the following JSON (call id will likely be different but valid eg. some integer.
```JSON
{
    "message": "call status updated to IN_PROGRESS",
    "call": {
        "id": 2,
        "status": "IN_PROGRESS"
    }
}
```
8. Hear call change from hold music.
